package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@itwin/synchronization-report-react",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
no dev version?

Reply:
no, this is a direct patch to stable main version